diff --git a/node_modules/react-native-paper/src/components/SegmentedButtons/SegmentedButtonItem.tsx b/node_modules/react-native-paper/src/components/SegmentedButtons/SegmentedButtonItem.tsx
index d88f2a4..80e1d48 100644
--- a/node_modules/react-native-paper/src/components/SegmentedButtons/SegmentedButtonItem.tsx
+++ b/node_modules/react-native-paper/src/components/SegmentedButtons/SegmentedButtonItem.tsx
@@ -251,7 +251,7 @@ const SegmentedButtonItem = ({
 const styles = StyleSheet.create({
   button: {
     flex: 1,
-    minWidth: 76,
+    minWidth: 0,
     borderStyle: 'solid',
   },
   label: {
diff --git a/node_modules/react-native-paper/src/components/Snackbar.tsx b/node_modules/react-native-paper/src/components/Snackbar.tsx
index 2d552bb..b70c226 100644
--- a/node_modules/react-native-paper/src/components/Snackbar.tsx
+++ b/node_modules/react-native-paper/src/components/Snackbar.tsx
@@ -334,6 +334,7 @@ const Snackbar = ({
                 accessibilityRole="button"
                 borderless
                 onPress={onIconPress}
+                containerColor={theme.colors.inverseSurface}
                 iconColor={theme.colors.inverseOnSurface}
                 rippleColor={rippleColor}
                 theme={theme}
